kafka-connect-s3/src/main/java/io/confluent/connect/s3/format/KeyValueHeaderRecordWriterProvider.java

```diff
@@ -88,17 +88,13 @@ public void write(SinkRecord sinkRecord) {
 
         // keyWriter != null means writing keys is turned on
         if (keyWriter != null && sinkRecord.key() == null) {
-          throw new DataException(
-              String.format("Key cannot be null for SinkRecord: %s", sinkRecord)
-          );
+          throw new DataException("Key cannot be null for SinkRecord.");
```

Comment:
Maybe we can include the topic name for better debuggability?

Reply:
ack